h2o-r/h2o-package/R/predict.R

```diff
@@ -0,0 +1,49 @@
+#'
+#' H2O Prediction from R
+#'
+#' Provides the method h2o.predict with which you can predict a MOJO or POJO Jar model
+#' from R.
+#'
+
+
+
+#' @param model  String with file name of MOJO or POJO Jar
+#' @param json  JSON String with inputs to model
+#' @param classpath  (Optional) Extra items for the class path of where to look for Java classes, e.g., h2o-genmodel.jar
+#' @return Returns an object with the prediction result
+#' @importFrom jsonlite fromJSON
+#' @examples
+#' \donttest{
+#' library(h2o)
+#' h2o.predict_json('~/GBM_model_python_1473313897851_6.zip', '{"C7":1}')
+#' h2o.predict_json('~/GBM_model_python_1473313897851_6.zip', '{"C7":1}', c(".", "lib"))
+#' }
+#' @usage
+#' h2o.predict_json(model, json, classpath)
+#' @name h2o.predict_json
+
+h2o_predict_json_java <- "java"
+h2o_predict_json_javaopts <- "-Xmx4g"
+h2o_predict_json_javapath <- c( c(".", "h2o-genmodel.jar"),  sapply(.libPaths(), paste0, "/h2o/java/h2o.jar", USE.NAMES=FALSE) )
```

Comment:
`/h2o/java/h2o.jar` looks platform specific, we have `.h2o.downloadJar` which will detect path to h2o.jar. Better to not duplicate functionality which already exists.